test/externalTests/ens.sh

```diff
@@ -27,34 +27,32 @@ source test/externalTests/common.sh
 verify_input "$1"
 export SOLJSON="$1"
 
-function compile_fn { npx truffle compile; }
-function test_fn { npm run test; }
+function compile_fn { yarn build; }
+function test_fn { yarn test; }
 
 function ens_test
 {
-    local repo="https://github.com/ensdomains/ens.git"
+    local repo="https://github.com/ensdomains/ens-contracts.git"
     local branch=master
```

Comment:
Oh. You mean an upstream tag or a full-blown fork in solidity-external-tests?

Reply:
Upstream tag, or even just commit hash.

Reply:
Looks like the tag I used (`v0.0.5`) is too ancient and lots of tests fail. Switching to a specific commit.

Reply:
Looks like `git clone` does not work with specific commits. The only thing I could do is to clone the whole repo (without `--depth 1`) and then check out the commit. I'd need to add a separate function to handle that case. I can do it but it's a bit of a hassle. How unstable is their `master` really?

Reply:
So what do we do about it? Can it stay at `master` after all?

Reply:
There's a new upstream tag dated yesterday: https://github.com/ensdomains/ens-contracts/tags

Reply:
Oh, nice. I even checked the repo today but I missed it because I only looked at releases.

In that case I'll switch to that and then let's get this merged :)